delfin/drivers/hpe/hpe_msa/ssh_handler.py

```diff
@@ -88,35 +88,34 @@ def list_storage_disks(self, storage_id):
             disk_info = self.ssh_pool.do_exec('show disks')
             disk_detail = self.handle_xml_to_json(disk_info, 'drives')
             disks_arr = []
-            if disk_detail:
-                for data in disk_detail:
-                    health = data.get('health')
-                    status = constants.StoragePoolStatus.OFFLINE
-                    if health == 'OK':
-                        status = constants.StoragePoolStatus.NORMAL
-                    size = self.parse_string_to_bytes(data.get('size'))
-                    physical_type = consts.DiskPhysicalType.\
-                        DISK_PHYSICAL_TYPE.get(data.get('description'),
-                                               constants.DiskPhysicalType.
-                                               UNKNOWN)
-                    rpm = data.get('rpm')
-                    if rpm:
-                        rpm = int(rpm) * consts.RpmSpeed.RPM_SPEED
-                    data_map = {
-                        'native_disk_id': data.get('location'),
-                        'name': data.get('location'),
-                        'physical_type': physical_type,
-                        'status': status,
-                        'storage_id': storage_id,
-                        'native_disk_group_id': data.get('disk-group'),
-                        'serial_number': data.get('serial-number'),
-                        'manufacturer': data.get('vendor'),
-                        'model': data.get('model'),
-                        'speed': rpm,
-                        'capacity': int(size),
-                        'health_score': status
-                    }
-                    disks_arr.append(data_map)
+            for data in disk_detail:
+                health = data.get('health')
+                status = constants.StoragePoolStatus.OFFLINE
+                if health == 'OK':
+                    status = constants.StoragePoolStatus.NORMAL
+                size = self.parse_string_to_bytes(data.get('size'))
+                physical_type = consts.DiskPhysicalType.\
+                    DISK_PHYSICAL_TYPE.get(data.get('description'),
+                                           constants.DiskPhysicalType.
+                                           UNKNOWN)
+                rpm = data.get('rpm')
+                if rpm:
+                    rpm = int(rpm) * consts.RpmSpeed.RPM_SPEED
+                data_map = {
+                    'native_disk_id': data.get('location'),
+                    'name': data.get('location'),
+                    'physical_type': physical_type,
+                    'status': status,
+                    'storage_id': storage_id,
+                    'native_disk_group_id': data.get('disk-group'),
+                    'serial_number': data.get('serial-number'),
+                    'manufacturer': data.get('vendor'),
+                    'model': data.get('model'),
+                    'speed': rpm,
+                    'capacity': int(size),
```

Comment:
Do you need to switch again here
 'capacity': int(size),

Reply:
There are decimals in the return value